packages/global-styles/src/hooks/use-sync-global-styles-user-config.ts

```diff
@@ -1,21 +1,20 @@
-// import { GlobalStylesContext } from '@wordpress/edit-site/build-module/components/global-styles/context';
-// import { mergeBaseAndUserConfigs } from '@wordpress/edit-site/build-module/components/global-styles/global-styles-provider';
 import { useContext, useEffect } from 'react';
+import { GlobalStylesContext, mergeBaseAndUserConfigs } from '../gutenberg-bridge';
 import type { GlobalStylesObject } from '../types';
 
 const useSyncGlobalStylesUserConfig = ( globalStyles: GlobalStylesObject[], enabled: boolean ) => {
-	// const { user, setUserConfig } = useContext( GlobalStylesContext );
-	// useEffect( () => {
-	// 	if ( ! enabled ) {
-	// 		return;
-	// 	}
-	// 	setUserConfig( () =>
-	// 		globalStyles
-	// 			.filter( Boolean )
-	// 			.reduce( ( prev, current ) => mergeBaseAndUserConfigs( prev, current ), {} )
-	// 	);
-	// }, [ globalStyles, enabled ] );
-	// return user;
+	const { user, setUserConfig } = useContext( GlobalStylesContext );
+	useEffect( () => {
+		if ( ! enabled ) {
```

Comment:
Note: this file changed on trunk, so it might need to be refreshed. (I resolved the merge conflict in the base PR to maintain compatibility with this branch)

Reply:
Rebased. Thanks for noting that!